src/components/navigation-bar/navigation-bar.pw.tsx

```diff
@@ -0,0 +1,313 @@
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NavigationBarComponent
+          offset={`${offset}px`}
+          orientation="top"
+          position="fixed"
+        />
+      );
+
+      await expect(navigationBar(page)).toHaveCSS("top", `${offset}px`);
+    });
+  });
+
+  offsetVal.forEach((offset) => {
+    test(`should render with orientation set to bottom and offset prop set to ${offset} px`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NavigationBarComponent
+          offset={`${offset}px`}
+          orientation="bottom"
+          position="fixed"
+        />
+      );
+
+      await expect(navigationBar(page)).toHaveCSS("bottom", `${offset}px`);
+    });
+  });
+
+  (["top", "bottom"] as const).forEach((orientation) => {
+    test(`should render NavigationBar component with orientation prop set to ${orientation}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <NavigationBarComponent orientation={orientation} position="fixed" />
+      );
+
+      await expect(navigationBar(page)).toHaveCSS(orientation, "0px");
+    });
+  });
+
+  (["fixed", "sticky"] as NavigationBarProps["position"][]).forEach(
+    (position) => {
+      test(`should render with position prop set to ${position} and work as expected`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <div
+            style={{
+              height: "250px",
+            }}
+          >
+            <NavigationBar
+              position={position}
+              orientation="top"
+              offset="25px"
+              aria-label="header"
+            >
+              <Box display="flex" flex="1" maxWidth="1000px" margin="0 auto">
+                <Menu display="flex" flex="1">
+                  <MenuItem flex="1" onClick={() => {}}>
+                    Menu Item One
+                  </MenuItem>
+                </Menu>
+              </Box>
+            </NavigationBar>
+            <div
+              style={{
+                height: "1000px",
+              }}
+            />
+            <NavigationBar
+              position={position}
+              orientation="bottom"
+              offset="25px"
+              aria-label="footer"
+            >
+              <Box display="flex" flex="1" maxWidth="1000px" margin="0 auto">
+                <Menu display="flex" flex="1">
+                  <MenuItem flex="1" onClick={() => {}}>
+                    Menu Item One
+                  </MenuItem>
+                </Menu>
+              </Box>
+            </NavigationBar>
+          </div>
+        );
+
+        await expect(navigationBar(page).nth(0)).toBeVisible();
+
+        await expect(navigationBar(page).nth(1)).toBeVisible();
+
+        await page.evaluate(() => window.scrollTo(1000, 1000));
+
+        if (position === "fixed") {
+          await expect(navigationBar(page).nth(0)).toBeVisible();
+        }
```

Comment:
I might be wrong but I think the top navigation bar should not be visible if its position is set to fixed. This may be passing because ```toBeVisible``` finds elements that are not onscreen but which are in the DOM. I think ```toBeInViewport``` does what the test requires. 

There should also be an ```else``` assertion to verify that the top navigation bar is visible when its position is set to sticky.